### ISSUE TITLE
Call to undefined function seregazhuk\class_uses_recursive()

### ISSUE THREAD

**Issue by mashuble** [seregazhuk/php-pinterest-bot#320]
When I try to create a new Pinterest profile with the PHP Pinterest Bot:

I got this error:

> [Symfony\Component\Debug\Exception\FatalThrowableError]
> Call to undefined function seregazhuk\class_uses_recursive()

On the file seregazhuk/pinterest-bot/src/Api/Providers/Core/Provider.php:124

Is it just me ? Did I do something wrong ?

here is my code 
```
$registration = new Registration(
     'my-email',
     'my-password',
     'My Name'
 );

$registration
     ->setAge(32)
     ->setCountry('US')
     ->setFemaleGender()
     ->setSite('https://google.com');

$bot->auth->registerBusiness($registration);
```

**Comment by seregazhuk** [comment]
Hi, @mashuble 
I think, I've found the problem. Working on the solution. Are you using Laravel?

**Comment by seregazhuk** [comment]
Fixed, please update to version `5.5.7`

**Comment by mashuble** [comment]
Ok thanks
